### ISSUE TITLE
How to enable fiber in stable version?

### ISSUE THREAD

**Issue by lnwu** [facebook/react#11926]
I searched a lot, but can't find any messages to enable this feature in stable version.

**Comment by vikramcse** [comment]
@lnwu React Fiber reconciler is already enabled in react version 16 and above. You do not need to specify any flag to enable it.

**Comment by aweary** [comment]
@vikramcse is correct, Fiber is not something that is enabled or disabled, it's a description of how the reconciler is architectured.

**Comment by lnwu** [comment]
I mean async rendering actually, i did not see any change in the way of rendering in version 16.2.0.

**Comment by aweary** [comment]
@lnwu async rendering is still a work in progress!

**Comment by mindplay-dk** [comment]
> async rendering is still a work in progress!

@aweary Still in 2019?

I can't seem to find any real documentation.

I've been trying to demo it myself - based on this [older demo](https://pomber.github.io/incremental-rendering-demo/react-async.html), where it does work, I created this [simplified demo](https://codesandbox.io/s/suspicious-rosalind-i06mv) using React, and some React-likes, as I wanted to compare the results.

The animation glitch is just as visible with React as it is with the others.

Maybe it doesn't work with hooks or functional components?

Or is there something crucial I've overlooked in the original example?
